osidb/tests/test_tracker.py

```diff
@@ -56,6 +57,26 @@ def test_aggregeted_impact(
         tracker = TrackerFactory(affects=[affect1, affect2], embargoed=flaw1.embargoed)
         assert tracker.aggregated_impact == expected_impact
 
+    def test_is_unacked(self):
+        """
+        test that (un)acked property works correctly
+        """
+        ps_module = PsModuleFactory()
+        acked_ps_update_stream = PsUpdateStreamFactory(
+            ps_module=ps_module,
+            unacked_to_ps_module=None,
+        )
+        unacked_ps_update_stream = PsUpdateStreamFactory(
+            ps_module=ps_module,
+            unacked_to_ps_module=ps_module,
+        )
+
+        acked_tracker = TrackerFactory(ps_update_stream=acked_ps_update_stream.name)
+        unacked_tracker = TrackerFactory(ps_update_stream=unacked_ps_update_stream.name)
+
+        assert acked_tracker.is_acked and not acked_tracker.is_unacked
+        assert not unacked_tracker.is_acked and unacked_tracker.is_unacked
```

Comment:
Could this be simplified? From the logic of `is_acked` and `is_unacked` properties it kinda feels like testing `True and not False` statement :grinning:

Reply:
I know that the model methods are defined in the way that only one contains the logic and the other only reverts the first one but I think that the test should not be aware of it and really check whether both give the correct results.